src/config/env/production.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    env: 'production',
+    db: 'mongodb://localhost/db',
+    port: process.env.SH_BACKEND_PORT || 3000,
+    host: '0.0.0.0',
+    seedDB: true,
+    staticPath: process.env.SH_PATH_FRONTENT_DIST,
```

Comment:
Maybe better to use ```process.env.SH_PATH_FRONTENT_DIST``` in needed place, because in other configs this is not needed or this need to be added to all configs files.